src/controller/userController.ts

```diff
@@ -3,10 +3,22 @@ import * as UserService from '../services/userService'
 
 export const getAllUsers = async (_req: Request, res: Response): Promise<void> => {
   const allUser = await UserService.getAllUsers()
+  if ('code' in allUser) {
```

Comment:
Please, use object destructiring...

`const { code } = allUser`

Good practice is avoid Nested IF_ELSE statements, people really hate.
maybe write..

```
// check is code exist and is 404
if (code && code === 404 ) {
  res.status(404)
  // You forgot to send object message 
  res.send( NEED TO CREATE ONE OBJECT HERE. Something generic which suits allUsers, allVenues, users by ID and venues by ID)
  }
//no need else. Because if is false, always goes here.
res.status(200)
res.send(allUsers)
```

Reply:
Cleaned up controllers a bit from refactor.